app/src/test/java/org/simple/clinic/bp/entry/BpValidatorTest.kt

```diff
@@ -0,0 +1,72 @@
+package org.simple.clinic.bp.entry
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class BpValidatorTest {
```

Comment:
Can we make this a parameterized test and add a couple more test cases to each test?

Reply:
Good idea, we should do this. I'll do this in a separate PR, again because this change wasn't a part of this feature. I'm adding these TODOs to https://github.com/simpledotorg/simple-android/pull/465's description but will move them to their pivotal stories in the end.